app/service/auth.py

```diff
@@ -0,0 +1,65 @@
+import bcrypt, jwt
+
+from sqlmodel import Session, select
+from fastapi import FastAPI, status
+from fastapi.exceptions import HTTPException
+from typing import Optional
+from jose import JWTError
+
+from models.models import User, LoginRequest, TokenData
+from db import engine, SECRET_KEY, ALGORITHM
+
+app = FastAPI
+
+def login_for_access_token(login_request: LoginRequest) -> dict:
+    user = _authenticate_user(email=login_request.email, password=login_request.password)
```

Comment:
```suggestion
def get_access_token_after_login_success(login_request: LoginRequest) -> dict:
    user = _authenticate_user(email=login_request.email, password=login_request.password)
```
p2: 문법에 맞는지는 조금 헷갈리는데 의미를 좀 더 명확하게 하면 좋을것 같습니다. 생각해 보시고 더 좋은 네이밍을 찾아보셔도 좋을것 같아요.

Reply:
네! 수정하였습니다!